system-tests/projects/svelte-webpack-unconfigured/README.md

```diff
@@ -0,0 +1,64 @@
+# svelte app
```

Comment:
Let's remove this file and the `.gitignore` to reduce the file bloat. This goes for both the vite and webpack app, so any files that aren't actively used should be removed.